app/bundles/FormBundle/Model/SubmissionModel.php

```diff
@@ -225,7 +225,7 @@ public function saveSubmission($post, $server, Form $form, Request $request, $re
                 $params = $components['fields'][$f->getType()];
                 if (!empty($value)) {
                     if (isset($params['valueFilter'])) {
-                        if (is_string($params['valueFilter']) && is_callable(['\Mautic\CoreBundle\Helper\InputHelper', $params['valueFilter']])) {
+                        if (is_string($params['valueFilter']) && is_callable(['\\'.\Mautic\CoreBundle\Helper\InputHelper::class, $params['valueFilter']])) {
```

Comment:
Is this correct?
```suggestion
                        if (is_string($params['valueFilter']) && is_callable([\Mautic\CoreBundle\Helper\InputHelper::class, $params['valueFilter']])) {
```

Reply:
You're right, this could be improved, I'll check :+1: